modules/backing-up-storage-data-from-amazon-ebs.adoc

```diff
@@ -0,0 +1,68 @@
+:_module-type: PROCEDURE
+
+[id='backing-up-storage-data-from-amazon-ebs_{context}']
+= Backing up storage data from Amazon EBS
+
+[role='_abstract']
+ifndef::upstream[]
+Red Hat recommends that you back up the data on your persistent volume claims (PVCs) regularly. 
+endif::[]
+ifdef::upstream[]
+It is good practice to back up the data on your persistent volume claims (PVCs) regularly. 
+endif::[]
+Backing up your data is particularly important before deleting a user and before uninstalling {productname-short}, as all PVCs are deleted when {productname-short} is uninstalled.
+
+.Prerequisites
+* You have credentials for OpenShift Cluster Manager (link:https://console.redhat.com/openshift/[https://console.redhat.com/openshift/]).
```

Comment:
Does this apply upstream? I don't know, so it might be an SME question.

Reply:
yes it does - I checked with SME: "The "backing up" sections are not RHODS/ODH specific, so they would apply equally to both."